C4-Documentation/c4-code-finance_feedback_engine-agent.md

```
@@ -156,15 +155,12 @@ def __init__(
 
 ### Public Methods
 
-#### `supports_signal_only_mode() -> bool`
-**Location**: `trading_loop_agent.py:147-170`
+#### `supports_signal_only_mode() -> bool` *(removed / legacy)*
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)